src/components/pages/types.ts

```diff
@@ -1,4 +1,13 @@
-export interface PageMainProps {
-  cardCount: number,
-  placesToStay: number,
+import { Hotel } from '../../data-type';
+
+export interface MainPageProps {
+  hotelsData: Array<Hotel>,
+}
+
+export interface FavoritesPageProps {
```

Comment:
То что определен отдельный интерфейс для компонента - очень разумно. Кто его знает, как они будут в дальнейшем развиваться. 

```suggestion
     export type FavoritesPageProps  = MainPageProps;
```

А вот такой подход подсказывает читателю, что хотя мы и ожидаем изменения пропсов, сейчас они суть одно и то же.